### PR TITLE
Fix/onboarding layout ui

### PR DESCRIPTION
**Pull request by odb366**
This PR contains UI fixes for the onboarding process:
- Added borders to all input fields. 
- Adjusted layout and form spacing for more consistency between screens.
- Highlighted the next button and increased its click area

onboarding-layout-fix2
onboarding-layout-fix1

### CODE REVIEW

**Comment by JimmyMow** [comment]
@odb366 Thanks for the PR.

I like all the changed except the gold hover on the `Next` button. Let's change it to be the same color scheme as the `Unlock` button within the `Login` component.

`background: $darkspaceblue;`

hover: `background: lighten($darkspaceblue, 10%);`

Can you make that change and resolve the merge conflicts? Then we should be good to merge.

**Comment by odb366** [comment]
@JimmyMow No probs man. 
The button color is changed according to recommendations and the merge conflict is resolved.
However, there are some eslint errors which I don't know how to fix:

screenshot 64

File: app/components/Onboarding/FormContainer.js [hunk 1]
```diff
@@ -7,7 +7,13 @@ import { FaAngleLeft, FaAngleRight } from 'react-icons/lib/fa'
 import zapLogo from 'icons/zap_logo.svg'
 import styles from './FormContainer.scss'
 
-const FormContainer = ({ title, description, back, next, children }) => (
+const FormContainer = ({
```
Comment (JimmyMow) [inline app/components/Onboarding/FormContainer.js]: Change this back to 1 line, no need for line break here

File: app/components/Onboarding/FormContainer.js [hunk 2]
```diff
@@ -23,20 +29,22 @@ const FormContainer = ({ title, description, back, next, children }) => (
...
+    <div className={styles.content}>
+      {children}
+    </div>
 
     <footer className={styles.footer}>
       <div className={styles.buttonsContainer}>
         <section>
           {back && (
-            <div onClick={back}>
+            <div onClick={back} className={styles.backButton} >
```
Comment (JimmyMow) [inline app/components/Onboarding/FormContainer.js]: extra space before `>` not needed

File: app/components/Onboarding/FormContainer.js [hunk 3]
```diff
@@ -23,20 +29,22 @@ const FormContainer = ({ title, description, back, next, children }) => (
...
               <FaAngleLeft style={{ verticalAlign: 'top' }} /> Back
             </div>
           )}
         </section>
         <section>
           {next && (
-            <div onClick={next}>
+            <div onClick={next} className={styles.nextButton} >
```
Comment (JimmyMow) [inline app/components/Onboarding/FormContainer.js]: extra space before `>` not needed

File: app/components/Onboarding/FormContainer.js [hunk 4]
```diff
@@ -46,6 +54,7 @@ const FormContainer = ({ title, description, back, next, children }) => (
   </div>
 )
 
+
```
Comment (JimmyMow) [inline app/components/Onboarding/FormContainer.js]: Remove this extra line

File: app/components/Onboarding/FormContainer.js [hunk 5]
```diff
@@ -23,20 +29,22 @@ const FormContainer = ({ title, description, back, next, children }) => (
       <p>{description}</p>
     </div>
 
-    <div className={styles.content}>{children}</div>
+    <div className={styles.content}>
```
Comment (JimmyMow) [inline app/components/Onboarding/FormContainer.js]: No need to break this out into multiple lines. Change this back to one line

**Comment by mrfelton** [comment]
We might want to add some more notes to https://github.com/LN-Zap/zap-desktop/blob/master/CONTRIBUTING.md about the lint setup. If users have things properly setup in their local environment, all lint/formatting errors should be automatically resolved on save and/or on commit.

Steps a user should take to get things setup properly include

1. installing the Prettier addon for your text editor of choice (see https://prettier.io/)
2. enabling our pre-commit hooks:
```bash
echo commit-msg >> .opt-in
echo pre-commit >>  .opt-in
```

With both of those steps in place users will get:

1. Automatic reformatting of files with prettier on save
2. Automatic reformatting off changed files with prettier on commit
3. Automatic lint of changed files on commit (will not allow commit with files that fail lint)
3. Validation of commit message format (will not allow commit with invalid commit message)

**Comment by mrfelton** [comment]
@odb366 - based on your screenshot - it looks like your branch is not up to date since in your screenshot there is a reference to `babili-webpack-plugin` which is no longer in use. I'd suggest you rebase your branch off master, remove your node_modules directory, and then run `yarn` to get a fresh install.

Also, follow the notes in my previous message if you want an enhanced development workflow with automatic code formatting and linting.

**Comment by JimmyMow** [comment]
Tested ACK 273ec2b

**Pull request merged**